src/components/Filter/Filter.js

```diff
@@ -4,9 +4,13 @@ import Button from "components/Button/Button";
 import "./_filter.scss";
 
 const Filter = ({ label, filters }) => {
+  const _handleFilter = () => {
+    console.log("Handle filter click");
+  };
+
   const filterButtons = filters.map(filter => (
-    <span key={filter} className="p-filter__item">
-      <Button>{filter}</Button>
+    <span key={filter} className="p-filter__item 1">
+      <Button onClick={() => _handleFilter()}>{filter}</Button>
```

Comment:
Using this will not pass the click event through to the `_handleFilter` call....you'll want...

```suggestion
      <Button onClick={_handleFilter}>{filter}</Button>
```